refactor_me.py

```diff
@@ -2,25 +2,49 @@
 
 from collections import namedtuple, defaultdict
 from operator import itemgetter
+from sys import stdout
+import io
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', ('category', 'amount'))
 
 
 def sum_expenses(expenses, min_amount=0):
+    """Aggregates all Expense entries into a categorized list of sums
```

Comment:
"Aggregates all" -> either rename function to "aggregate_expenses" or use "Sum all". There is no reason to use synonymical or contradictive words in function name vs description

Reply:
Fixed in a new commit